com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/MyLanguageServerImpl.java

```diff
@@ -69,13 +68,22 @@
   @Override
   public void initialized(@Nullable InitializedParams params) {
     LanguageClient client = clientProvider.get();
-    Registration registration =
-        new Registration("copybooksWatcher", "workspace/didChangeWatchedFiles", createWatcher());
-    RegistrationParams registrationParams =
-        new RegistrationParams(Collections.singletonList(registration));
+    List<Registration> registrationList = new ArrayList<>();
```

Comment:
Also here, it is better to add elements to the list in this method, not passing the list to the delegated methods